react/ui-components/src/molecules/stories/FormCard.stories.js

```
@@ -95,9 +95,7 @@ export const SimpleLayout = () => {
     width: isMobileView ? "100%" : "30%",
   };
 
-  const headerData = [
-    <TextBlock header={"Enter Details"}></TextBlock>
-  ]
+  const headerData = [<TextBlock header={"Enter Details"}></TextBlock>];
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)